src/systempolicy/systemPolicy.go

```diff
@@ -1464,6 +1464,17 @@ func StartSystemLogRcvr() {
 	for {
 		if cfg.GetCfgSystemLogFrom() == "kubearmor" {
 			url := cluster.GetKubearmorRelayURL()
+			if url == "" {
+				log.Error().Msg("kubearmor-relay url not found, retrying...")
+				for i := 0; i < types.Maxtries; i++ {
+					time.Sleep(10 * time.Second)
+					url = cluster.GetKubearmorRelayURL()
+					if url != "" {
+						break
+					}
+				}
+				return
```

Comment:
are we sure we want to return here 
doing this would mean if we install kubearmor after the 6 retries the discovery engine wont be able to connect to the relay server